tools/Microsoft.Health.Tools.Sql.Tasks/Sql.targets

```diff
@@ -28,6 +28,7 @@
             SqlScript="@(SqlScript)" />
     </Target>
 
-    <UsingTask TaskName="GenerateFullScript" AssemblyFile="netstandard2.0\Microsoft.Health.Tools.Sql.Tasks.dll" />
+    <UsingTask TaskName="GenerateFullScript" AssemblyFile="netstandard2.1\Microsoft.Health.Tools.Sql.Tasks.dll" Condition=" '$(MSBuildRuntimeType)' == 'Core' "/>
```

Comment:
Probably more appropriate to check would be `'$(MSBuildRuntimeType)' == 'Full'` as I think .NET Standard 2.1 should work with .NET Core/.NET or Mono

Reply:
Thanks, I will fix it now.

Reply:
I have tried it according to [MSBuildRuntimeType ](https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-reserved-and-well-known-properties?view=vs-2022#:~:text=Full%20indicating%20that%20MSBuild%20is%20running%20on%20the%20desktop%20.NET%20Framework%2C%20Core%20indicating%20that%20MSBuild%20is%20running%20on%20.NET%20Core%20(for%20example%20in%20dotnet%20build)%2C%20or%20Mono%20indicating%20that%20MSBuild%20is%20running%20on%20Mono.)and found out that it will work if we use 'Full' to specify .NET Framework and use 'Core' and 'Mono' to specify .NET Core and Mono.